practitioner-service/src/main/java/uk/ac/ox/ndph/arts/practitioner_service/exception/ArgumentException.java

```diff
@@ -0,0 +1,14 @@
+package uk.ac.ox.ndph.arts.practitioner_service.exception;
+
+import org.springframework.http.HttpStatus;
+
+public class ArgumentException extends HttpStatusException{
```

Comment:
For all the exceptions that should be mapped to custom http status codes, you can use spring 'ResponseStatus' annotation: @ResponseStatus(value = HttpStatus. BAD_REQUEST)
Spring's REST service will automatically resolve the exception and map to a response with the relevant http status code.   
This can also simplify each exception class, that now can extend RuntimeException directly.